hw/top_earlgrey/ip/ast/ast.core

```diff
@@ -107,6 +107,7 @@ targets:
     default_tool: verilator
     parameters:
       - SYNTHESIS=true
+      - AST_BYPASS_CLK=true
```

Comment:
@msfschaffner 
i think i need some reminder... i believe @msfschaffner removed this as part of an earlier PR.

Reply:
You may have removed it, but since I was not aware of that the source had it. Again please avoid changes to the AST code without notifying me.
The AST code for the OS is generated and not the source.